prot.c

```diff
@@ -932,7 +938,8 @@ fmt_stats(char *buf, size_t size, void *x)
             srv->wal.nrec,
             srv->wal.filesize,
             id,
-            node_info.nodename);
+            node_info.nodename,
+            srv->draining);
```

Comment:
Fixed in 90950db6c378febdf12326898cfbba65def0cb23. Let me know if you need anything else here.